pkg/appliance/upgrade_status.go

```diff
@@ -67,6 +67,10 @@ func (u *UpgradeStatus) upgradeStatus(ctx context.Context, appliance openapi.App
 				tracker.Update(s)
 			}
 			if util.InSlice(s, undesiredStatuses) {
+				if tracker != nil {
+					// send error details for tracker
+					tracker.Update(details)
```

Comment:
should it be a concat string of  `s + details` maybe? how does this look in practice?

Reply:
It looks like this:
```bash
[...]
[2022-08-26T08:09:48+02:00] Uploading upgrade image:
 ✓ appgate-6.0.1-29609-beta.img.zip: upload complete

[2022-08-26T08:11:06+02:00] Preparing image on appliances:
 ⨯ controller-d7e397ee-b8c6-4829-82bf-5ebe22ff1faf-site1: failed - Signature incorrect
 ⨯ gateway-d7e397ee-b8c6-4829-82bf-5ebe22ff1faf-site1: failed - Signature incorrect
2 errors occurred:
	* Upgrade failed on controller-d7e397ee-b8c6-4829-82bf-5ebe22ff1faf-site1 - Signature incorrect
	* Upgrade failed on gateway-d7e397ee-b8c6-4829-82bf-5ebe22ff1faf-site1 - Signature incorrect
```
There's a concat in the tracker code, so the status is always going to be "failed" and then the reason is going to get concatenated like `"failed" + " - " + reason`. Thinking of it now, it probably makes more sense to concatenate it here though, since the status isn't always going to be "failed".